docker-compose.yml

```diff
@@ -1,14 +1,13 @@
-version: '3.8'
-
 services:
   postgres:
-    image: postgres:17
+    image: postgres:${POSTGRES_VERSION:-17}
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
-      POSTGRES_DB: postgres_doctrine_test
+      POSTGRES_USER: ${POSTGRES_USER:-postgres}
+      POSTGRES_PORT: ${POSTGRES_PORT:-5432}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres}
+      POSTGRES_DB: ${POSTGRES_DB:-postgres_doctrine_test}
     ports:
-      - "5432:5432"
+      - "${POSTGRES_PORT:-5432}:${POSTGRES_PORT:-5432}"
```

Comment:
This was the real thing I wanted to add. I am running another Postgres server at that port and didn't want to turn it off just to run the tests for this package.